src/module-1/weirdString/index.js

```diff
@@ -1 +1,11 @@
-export const weirdString = (str = "") => {};
+export const weirdString = (str = "") => {
+   let text = str
+   const result = text.toUpperCase().split(" ").map(word => {
+      const len = word.length;
+      if (len > 1) {
+         word = word.substring(0, len - 1) + word[len - 1].toLowerCase();
```

Comment:
function arguments mutation is a bad practice
so the new variable should be created here